lib/rules/prefer-rest-params.js

```diff
@@ -32,6 +32,28 @@ function getVariableOfArguments(scope) {
     return null;
 }
 
+/**
+ * Checks if the given reference is not normal member access.
+ *
+ * - arguments         .... true    // not member access
+ * - arguments[i]      .... true    // computed member access
+ * - arguments[0]      .... true    // computed member access
+ * - arguments.length  .... false   // normal member access
```

Comment:
This will warn `arguments["length"]` since this distinguishes by computed or not. We can avoid warning by `arguments.length`, so I thought this does not need a complex check (e.g. `arguments["0"]` is same as `arguments[0]`).